src/blockrelay/graphene.h

```diff
@@ -46,16 +47,27 @@ class CMemPoolInfo
 
 class CGrapheneBlock
 {
+private:
+    uint64_t nonce;
+
 public:
+    mutable uint64_t shorttxidk0, shorttxidk1;
```

Comment:
In C++11 the 'mutable' keyword is defined to be about threading, whereas before it was about const-ness. A mutable variable is supposed to be hread-safe. A simple 64 bit int is not thread safe and as such this mutable keyword is a red-flag on a misdesign somewhere (assuming it doesn't compile by simply removing the keyword).

Reply:
Thanks for the feedback. I removed the mutable keyword.